src/txn/txn.c

```diff
@@ -138,6 +138,50 @@ __wt_txn_release_snapshot(WT_SESSION_IMPL *session)
     }
 }
 
+/*
+ * __wt_txn_in_global_list --
+ *     Check if a transaction is still in the global transaction list. If it is not, then it is
+ *     committed, prepared, or rolled back. It is possible that we race with commit, prepare or
+ *     rollback and a transaction is still in the global list before the start of the call is
+ *     reported not in the list.
+ */
+bool
+__wt_txn_in_global_list(WT_SESSION_IMPL *session, uint64_t txnid)
+{
```

Comment:
I named it as __wt_txn_in_global_list because it is not a reliable check. Name it as __wt_txn_uncommitted would be misleading.